modules/sharedaddy/sharing.js

```diff
@@ -4,7 +4,7 @@ if ( sharing_js_options && sharing_js_options.counts ) {
 	var WPCOMSharing = {
 		done_urls : [],
 		get_counts : function() {
-			var url, urls, id, service, service_url;
+			var https_url, http_url, url, requests, id, service, service_request;
```

Comment:
`https_url` and `http_url` were removed in b0b68a2c6b62f627191d567759234153e1e1d463 and shouldn't be re-added here.